index.html

```diff
@@ -1238,6 +1238,192 @@
 <section id="converting-html-to-other-formats">
 <h2>Converting Microdata to other formats</h2>
 
+<section id="jsonLD">
+<h3>JSON-LD</h3>
+
+  <p>Given a list of nodes <var>nodes</var> in a <code>Document</code>, a user agent must
+  run the following algorithm to extract the Microdata from those nodes into a JSON-LD:</p>
```

Comment:
maybe change to "convert the Microdata from those nodes into a JSON-LD representation":